src/Sql/Sql/help/New-AzSqlElasticPool.md

```diff
@@ -53,6 +54,32 @@ StorageMB         : 409600
 Tags              :
 ```
 
+### Example 2: Create a vCore elastic pool
+
+```
+PS C:\> New-AzSqlElasticPool -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -ElasticPoolName "ElasticPool01" -Edition "GeneralPurpose" -vCore 2 -ComputeGeneration Gen5
+ResourceId          : /subscriptions/00000000-0000-0000-0000-000000000001/resourceGroups/ResourceGroup01/providers/Microsoft.Sql/servers/server01/elasticPools/ElasticPool01
+ResourceGroupName   : ResourceGroup01
+ServerName          : Server01
+ElasticPoolName     : ElasticPool01
+Location            : Central US
+CreationDate        : 8/29/2019 2:16:40 AM
+State               : Ready
+Edition             : GeneralPurpose
+SkuName             : GP_Gen5
+Dtu                 : 2
+DatabaseDtuMax      : 2
+DatabaseDtuMin      : 0
+Capacity            : 2
+DatabaseCapacityMin : 0
+DatabaseCapacityMax : 2
+Family              : Gen5
+MaxSizeBytes        : 34359738368
+StorageMB           : 32768
+Tags                : 
+```
+
+
 This command creates an elastic pool in the Standard service tier named ElasticPool01. The server
```

Comment:
@MashaMSFT you'll want to move this description to be in example 1's section and add a new description for this example that you've added to replace this.